operator/controllers/config/constants.go

```diff
@@ -68,6 +68,13 @@ const (
 	DriverVersion   = "2.5.0"
 	OperatorVersion = "2.5.0"
 
+	// Number of replica pods for CSI Sidecar deployment
+	ReplicaCount       = int32(2)
+	// Tolerations seconds for the CSI Sidecar deployment
+	TolerationsSeconds = int64(60)
+	// ContainerPort for /healthz/leader-election endpoint
+	AttacherLeaderLivenessPort    = int32(8080)
```

Comment:
This seems to be a very common port and might result in conflict with other user services running on same port on the node

Reply:
```
#oc describe pod ibm-spectrum-scale-csi-attacher-0 ibm-spectrum-scale-csi-attacher-1 | grep Liveness:
    Liveness:       http-get http://:http-endpoint/healthz/leader-election delay=10s timeout=10s period=20s #success=1 #failure=1
    Liveness:       http-get http://:http-endpoint/healthz/leader-election delay=10s timeout=10s period=20s #success=1 #failure=1

```
```
#oc describe pod ibm-spectrum-scale-csi-attacher-0 ibm-spectrum-scale-csi-attacher-1 | grep Port:
    Port:          6443/TCP
    Host Port:     0/TCP
    Port:          6443/TCP
    Host Port:     0/TCP
```
```
#oc get pods -o wide 
ibm-spectrum-scale-csi-attacher-0      1/1     Running   0          20s     10.254.12.45    worker0.csitest.cp.fyre.ibm.com   

ibm-spectrum-scale-csi-attacher-1      1/1     Running   0          17s     10.254.12.46    worker0.csitest.cp.fyre.ibm.com   
```

Attacher pods are scheduled in same node, using the same port in liveness probe. Liveness probe seems to be working fine without any port error.